### PR TITLE
Fixes the new items and adds the new combat armor to the PVT

### PR DESCRIPTION
**Pull request by BadAtThisGame302**
## About The Pull Request

Fixes the new item update items and adds the enclave combat armor to the PVT due to some vote winning

## Why It's Good For The Game

Cleans up code and the vote winning so 🤷🏻 here the new and improved armor for the PVT to fit them along the 10mm SMG

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
add: Added the new sprited combat armor/helmet to the PVT from Altum due to some vote he did winning
fix: fixed the new items, making them children of the intended armor and putting them in the correct .dms
fix: fixed the PVT loadouts buggin out yo!
rebalance: (Buffed the Hellfire helmet to the armor stats, since I forgot when I added it, shhh)
/:cl:

### CODE REVIEW

**Comment by Sinbest** [comment]
Please add images next time

File: code/modules/clothing/head/f13head.dm [hunk 1]
```diff
@@ -440,15 +440,6 @@
 	item_state = "advhelmet1"
 	armor = list("melee" = 80, "bullet" = 80, "laser" = 85, "energy" = 35, "bomb" = 72, "bio" = 100, "rad" = 100, "fire" = 90, "acid" = 50, "wound" = 90)
 
-/obj/item/clothing/head/helmet/f13/power_armor/advanced/hellfire
```
Comment (chemistrymain2) [inline code/modules/clothing/head/f13head.dm:443]: why is this being deleted

Comment (BadAtThisGame302) [reply code/modules/clothing/head/f13head.dm:443]: It's not... I moved it down.

**Comment by BadAtThisGame302** [comment]
> Please add images next time

Check the PR that actually added the sprites like 3 days ago man.

**Pull request merged**